### ISSUE TITLE
Progressive image loading

### ISSUE THREAD

**Issue by kyvaith** [DivanteLtd/vue-storefront#1202]
Just an idea - how about adopting Medium-syle progressive image loading? IMO it's the most elegant way to load images. Please look here: https://jmperezperez.com/medium-image-progressive-loading-placeholder/

**Comment by mercs600** [comment]
I totally agree with @filrak. It looks good, but this moment this is not required. Generally I think we should keep simple theme boilerplate which you can develop instead of implement ( and maintenance ) a lot of features.  But.. if you want, feel free to develop some plugin maybe ;-)

**Comment by kyvaith** [comment]
@mercs600 once again,
* I'm not a developer
* I posting stuff that seems to be for me must have in your project
* This are not requirements for near releases but for future improvement
* If they are reasonable for you just keep them, maybe with new tag like 'future-improvement'
*If they're not, just close.

Thanks, and thanks for your work and generally gre Idea for a project.
